src/App/Fossa/Analyze.hs

```diff
@@ -270,12 +308,12 @@ checkForEmptyUpload xs ys
   | xlen == 0 && ylen == 0 = NoneDiscovered
   | xlen == 0 || ylen == 0 = FilteredAll filterCount
   -- NE.fromList is a partial, but is safe since we confirm the length is > 0.
-  | otherwise              = FoundSome $ NE.fromList filtered
+  | otherwise = FoundSome $ NE.fromList filtered
   where
     xlen = length xs
     ylen = length ys
     filterCount = abs $ xlen - ylen
-    -- | Return the smaller list, since filtering cannot add projects
+
```

Comment:
Why was this comment removed?

Reply:
I'd bet this was an ormolu bug -- ormolu sometimes misbehaves when there are haddock-style comments (`-- |` / `-- ^`) where they aren't valid

Reply:
Yes, I didn't manually do this. I'll put this line back the way it was, thanks for pointing it out!

Reply:
Please also remove the pipe character so we don't run into this bug again.